### DISCUSSION TITLE
ANTLR discussion mailing list spam

### DISCUSSION THREAD

**Discussion by mwpowellhtx** [antlr/antlr4#4664] [General]
What are y'all doing about the spam landing in the discussion mailing list? Thank you...

**Comment by kaby76** [comment]
I'm glad someone else noticed. https://groups.google.com/g/antlr-discussion has been getting a spam post every few minutes for a week or two. I have been reporting at least twice a day. That sort of cleans up the archive for a few minutes. @parrt

**Comment by mwpowellhtx** [comment]
I do not have the bandwidth to chase that issue. The correct response is for the list owner to take action.

**Comment by parrt** [comment]
I don’t seem to be getting those spam messages. Can you send me one or two
so I can know who to look for? there are definitely some in purgatory that
I get notified about but I don’t let them through I let them fall off. They
all looked like Thai scripts

Dictation in use. Please excuse homophones, malapropisms, and nonsense.

**Reply by mwpowellhtx** [reply]
Just as a for instance.

![image](https://github.com/user-attachments/assets/90788839-171a-4f22-a05e-08ced4f3d9c1)

**Comment by parrt** [comment]
I see these pending but they should get ignored if not approved right?


***@***.*** ***@***.***)
แทงหวยลาว ซื้อหวยลาว บาทละ 1000 [ LINE: @cfm168 ] เว็บแทงหวยออนไลน์ จ่ายจริง
[image: ลาว.png] ลุ้นรวยทุกงวด! กับ "แทงหวยลาว" เว็บแทงหวยออนไลน์ จ่ายจริง บาทละ 1000 เหล่านักเสี่ย
Jul 21

แทงหวยออนไลน์ @cfm168 แทงหวยบาทละ 1000 เว็บหวย2024 ***@***.***)
แทงหวยออนไลน์ @cfm168 แทงหวยบาทละ 1000 เว็บหวย2024
[image: หวยออนไลน์.png] แทงหวยคลิ้ก : https://bit.ly/m/confirm-168 📌สอบถามเพิ่มเติม 𝐋𝐈𝐍𝐄 🆔: @
Jul 22

เว็บแทงบอล 2024 LINE: @cfm168 ***@***.***)
แทงบอลออนไลน์ LINE: @cfm168 ค่าน้ำ 4 ตังค์ เว็บแทงบอล พร้อมผลบอล
[image: แทงบอล.png] แทงบอลออนไลน์ เว็บใหญ่ จ่ายจริง เว็บแทงบอลที่มีความเป็นที่นิยมอันดับ 1 เชื่อถือ
Jul 22

***@***.*** ***@***.***)
🌼🌼ซื้อหวยลาวออนไลน์เว็บไหนดี 10 เว็บแทงหวยTGClubs จ่ายบาท 950 จ่ายจริง เชื่อถือได้ อันดับ1 ไม่มีเลขปิด🌼🌼
🌻🐝เว็บหวยลาว ซื้อหวยฮานอย เว็บไหนดี เว็บหวยออนไลน์ที่ดีที่สุดในไทย🌻🐝 ซื้อหวยออนไลน์เว็บไหนดี แท
Jul 24

แทงบอลออนไลน์ พนันบอล @cfm168 ***@***.***)
เว็บแทงบอลออนไลน์ แทงบอลออนไลน์อันดับ 1 @cfm168 ค่าน้ำ4ตังค์ ฝากขั้นต่ำ 20 บาท

**Comment by parrt** [comment]
changed this to reject from notify owners but we shouldn’t be seeing spam and i don’t in the incoming mail.

￼
